src/einsteinpy/tests/test_symbolic/test_tensor.py

```diff
@@ -40,6 +40,23 @@ def test_Tensor_getitem():
         assert obj[p, q, r] - test_list[p][q][r] == 0
 
 
+def test_Tensor_str():
+    x, y, z = symbols("x y z")
+    test_list = [[[x, y], [y, x]], [[z, x], [y, z]]]
+    obj1 = Tensor(test_list)
+
+    assert obj1.__str__ is not object.__str__
```

Comment:
I did not understand what is implied by this test. Maybe reply to this comment explaining what you tested?

Or change this taking this as reference.
https://github.com/einsteinpy/einsteinpy/blob/master/src/einsteinpy/tests/test_coordinates/test_coord_transform.py#L104-L123

Reply:
my approach to test if t the __str__ method is implemented is to check if the method is different from the default. I test it in my local machine and it worked

Reply:
Kindly capture the output of `print(object_name)`, and do some tests on it!

Reply:
![image](https://user-images.githubusercontent.com/51050291/60673926-3ccfe600-9e79-11e9-8b14-dd209be96a04.png)

As you can see in the screenshot, the output in the __str__ is human readable not like the output of object. What the test does it to check if the method __str__ of tensor class is not the default __str__ which I understand if it is not implemented, I would see something like "object x100..."

Reply:
`assert not "object at 0x" in str(obj1)`